app/views/food_items/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @foodItems.each do |i| %>
```

Comment:
underscores man. only underscores
camelcase - in js

use human-readable namings - `item` here